ledger-service/http-json-cli/src/main/scala/com/daml/http/OptionParser.scala

```diff
@@ -47,6 +49,8 @@ class OptionParser(getEnvVar: String => Option[String], supportedJdbcDriverNames
 
   override def renderingMode: RenderingMode = RenderingMode.OneColumn
 
+  private val KnownLogLevels = Set("ERROR", "WARN", "INFO", "DEBUG", "TRACE")
```

Comment:
As mentioned before, perfectly fine to not factor this out and share it with Sandbox in this PR but in that case, I recommend adding a `TODO` comment that you remove once you address it in the follow-up PR.

Reply:
already have the change which refactors this locally :sweat_smile:

Reply:
still added a todo